### DISCUSSION TITLE
Explore the idea of ​​connecting it with local LLMs

### DISCUSSION THREAD

**Discussion by peopleworks** [marcominerva/DatabaseGPT#52] [Ideas]
Have you thought about the idea of ​​connecting this wonder with local LLMS, since there are projects like Ollama that are super good and we can host them on our own servers.

**Comment by marcominerva** [comment]
If these projects expose an API like the one of OpenAI, the only thing to do is to update the [ChatGptNet](https://github.com/marcominerva/ChatGptNet) library (that is used by DatabaseGPT) to support Ollama (and others): https://github.com/marcominerva/ChatGptNet/blob/master/src/ChatGptNet/ServiceConfigurations/ChatGptServiceConfiguration.cs#L39
